presto-benchto-benchmarks/pom.xml

```diff
@@ -26,7 +26,7 @@
         <dependency>
             <groupId>com.teradata.benchto</groupId>
             <artifactId>benchto-driver</artifactId>
-            <version>0.3</version>
+            <version>0.4</version>
```

Comment:
what changed there?

Reply:
`profiles.directory` option. We can use multiple yaml files with benchmark configurations, e.g:
```
--profiles.directory=profile.yaml,profile-ovr.yaml
```